packages/boot/test/bootstrapTests/test-vat-orchestration.ts

```diff
@@ -60,18 +60,18 @@ test.before(async t => {
 
 test.after.always(t => t.context.shutdown?.());
 
-test('createAccount returns an ICA connection', async t => {
+test('makeAccount returns an ICA connection', async t => {
   const {
     runUtils: { EV },
   } = t.context;
 
   const orchestration = await EV.vat('bootstrap').consumeItem('orchestration');
 
-  const account = await EV(orchestration).createAccount(
+  const account = await EV(orchestration).makeAccount(
```

Comment:
Why does this need EV?

Reply:
[EV](https://github.com/Agoric/agoric-sdk/blob/a2c6d29539d15c981e82f689834f95bba5943d1a/packages/SwingSet/tools/run-utils.js#L57-L97) is a testing utility for bootstrap tests

Reply:
It's like `E` but works through RunUtils. https://github.com/Agoric/agoric-sdk/blob/231416ea1dfc952b8f360caa26c29812b590c9d3/packages/SwingSet/tools/run-utils.js#L55-L61